tickets/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from users.models   import User
+from flights.models import Flight
+
+class Passenger(models.Model): 
+    name_kor = models.CharField(max_length=45)
+    name_eng = models.CharField(max_length=45)
```

Comment:
```suggestion
    korean_name = models.CharField(max_length=45)
    english_name = models.CharField(max_length=45)
```
변수명은 줄임말 사용을 자제해 주세요.(위코드 컨벤션)

Reply:
수정하겠습니다!